Hi3Helper.Core/Lang/en.json

```diff
@@ -8,15 +8,15 @@
     "ChooseFolderDialogCancel": "Cancel",
     "ChooseFolderDialogPrimary": "Yes please!",
     "ChooseFolderDialogSecondary": "No, choose folder",
-    "ChooseFolderDialogSubtitle": "The default installation directory is set to:\r\n\r\n{0}\r\n\r\nDo you want to use this directory?",
+    "ChooseFolderDialogSubtitle": "The default installation directory is set to:\r\n\r\n{0}\r\n\r\nWould you like to use this directory?",
     "ChooseFolderDialogTitle": "Locate Folder",
     "FolderInsufficientPermission": "Permission Denied! Please choose another folder.",
     "FolderNotSelected": "No folder chosen. Please select a folder!",
-    "OverlayPrepareFolderSubtitle": "Applying permission...",
+    "OverlayPrepareFolderSubtitle": "Applying permission(s)...",
     "OverlayPrepareFolderTitle": "Preparing App Folder",
     "PageTitle": "First Startup Page",
     "Pg1LoadingSubitle1": "Preparing a list of supported games for you...",
-    "Pg1LoadingSubitle2": "Completed! Feel free to proceed!",
+    "Pg1LoadingSubitle2": "Completed! Moving on to the next step...!",
```

Comment:
Redundant, but `!` is unnecessary since we already use `...`

Reply:
Just wanted to add a bit more impact and to sound cheerful

Reply:
ah I see, its fine then <3